nameko/rpc.py

```diff
@@ -350,10 +351,16 @@ def __call__(self, *args, **kwargs):
         reply = self._call(*args, **kwargs)
         return reply.result()
 
-    def async(self, *args, **kwargs):
+    def call_async(self, *args, **kwargs):
         reply = self._call(*args, **kwargs)
         return reply
 
+    def async(self, *args, **kwargs):  # pragma: no cover
```

Comment:
You'll also need to use the `--rabbit-ctl-uri` flag, which specifies the management interface of the rabbit broker (you'll have to enable to management plugin too, if it's not already)